.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "CD++ Simulator"]
+	path = cdpp
+	url = https://github.com/SimulationEverywhere/CDPP_ExtendedStates-codename-Santi.git
```

Comment:
Should we add a fixed commit? Just in case breaking changes are made in that repo

Reply:
Maybe. I think git fixes the submodule to certain commit saved in the submodule directory. Will test that soon.

Reply:
Yes, it's fixed to a specific commit:
![image](https://user-images.githubusercontent.com/2617411/60460427-1d874d80-9c1a-11e9-8202-920468e67297.png)

Reply:
Yeah, I just googled it and that's the default git submodule behaivour. https://stackoverflow.com/questions/1777854/how-can-i-specify-a-branch-tag-when-adding-a-git-submodule?noredirect=1&lq=1